helm-chart/ray-cluster/values.yaml

```diff
@@ -141,13 +142,14 @@ worker:
   affinity: {}
   # Ray container security context.
   securityContext: {}
-  volumes:
-    - name: log-volume
-      emptyDir: {}
+  # Optional log volume and volumeMounts
```

Comment:
They are optional but make for a good default, so keeping them the same would be better.

Reply:
So then wouldn't we want them as defaults for the additional worker groups?

Reply:
I've re-enabled them as defaults and added them as a default for the `additionalWorkerGroups` but they aren't being merged now so if it gets overridden then the volumes/volumeMounts will no longer exist. OTOH, I think it may be smarter to make it an key that makes the default log volume/volumeMounts if it's true and not adding them if it's false. This way if someone overrides volumes but not volumeMounts you won't have a random volumeMount that has no volume.

Reply:
> I've re-enabled them as defaults and added them as a default for the `additionalWorkerGroups` ...

Adding default values for `additionalWorkerGroups` is a good idea. It would be better to add the default values in `values.yaml` instead of `raycluster-cluster.yaml`.

Reply:
The problem is that if the default values are in the `values.yaml` it's fine and all for the head node, the worker group, and the smallGroup example, but it makes them non-optional therefore requiring users to include a `volumeMounts: {}` key or copy/paste it to use the defaults. If they're expected defaults there is no reason for them not to exist within `raycluster-cluster.yaml`

Reply:
I've pushed up a new change that should handle the defaults sanely.

Reply:
Hmm, perhaps I should use "recommended configurations" instead of "default value" above.

Reply:
I think that the new changes with possibly some renaming should handle a lot of this idea though.